LogicForPad.java

```diff
@@ -0,0 +1,159 @@
+package CleanNotePad;
+
+import java.awt.Graphics;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.print.PageFormat;
+import java.awt.print.Printable;
+import java.awt.print.PrinterException;
+import java.awt.print.PrinterJob;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+
+import javax.swing.JFileChooser;
+import javax.swing.JFrame;
+import javax.swing.JMenu;
+import javax.swing.JMenuItem;
+import javax.swing.JOptionPane;
+import javax.swing.JTextPane;
+import javax.swing.text.Position;
+import javax.swing.text.StyledDocument;
+//This class holds all the business logic that the not pad needs to run
+public class LogicForPad extends JFrame{
+	//Array to hold the five most recent files
+	ArrayList<File> recentFiles=new ArrayList<File>();
+	
+	//Wipes the current file clean,lazy new file implementation
+    public void newFile(JTextPane fileContent) {
+    	fileContent.setText("");
+    }
+    

```

Comment:
great use of the JTextPane .copy() function, however I think the method name is somewhat confusing, I would have named it something like copyToClipboard